Gemfile.lock

```diff
@@ -130,7 +130,7 @@ GEM
       capistrano (~> 3.1)
     capistrano-passenger (0.2.0)
       capistrano (~> 3.0)
-    capistrano-rails (1.4.0)
+    capistrano-rails (1.5.0)
```

Comment:
### [capistrano-rails](https://github.com/capistrano/rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.0 → 1.5.0

[_[change-log](https://github.com/capistrano/rails/blob/master/CHANGELOG.md), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/capistrano/rails/compare/v1.4.0...v1.5.0).

These are the individual commits:
- (d4518cc) [Update README to discuss uploading the master.key (#232)](https://github.com/capistrano/rails/commit/d4518cc0428f812d1cf3d07d31c9579b588d52a6)
- (dec4805) [Add changelog_uri to metadata to easily link from rubygems.o…](https://github.com/capistrano/rails/commit/dec480554144bd1904f607dd665cf42c962f1e4b)
- (d86a8db) [handle restore of multiple asset manifests on rollback (#226…](https://github.com/capistrano/rails/commit/d86a8db16281f09d8cfff9ee791297134bce9801)
- (7d233fa) [Release 1.5.0](https://github.com/capistrano/rails/commit/7d233fa0975707f7489da892d4a05eb373976b1f)


</details>